### ISSUE TITLE
Heroku deploy issue

### ISSUE THREAD

**Issue by hvarday** [svet4/shipping-costs-sample#5]
The repository won't deploy on heroku when trying to deploy using the **button** link in the readme file, but it does deploy when manually cloned and deployed to heroku.

This button - <a href="https://heroku.com/deploy" target="_blank"><img src="https://www.herokucdn.com/deploy/button.svg"></a>

![image](https://cloud.githubusercontent.com/assets/11018069/26283329/0b1a2fae-3e44-11e7-826d-e73f04a9e1dd.png)

**Comment by geekvaibhav** [comment]
Yes! issue still persist and same issue is there in yahoo Weather API.

**Comment by birender-s** [comment]
Hi, i tried yesterday, my experience was bit different. Deployment gave success for this webhook but gives error when viewing details in Heroku. On the other hand, yahoo weater API worked perfectly fine, i could test webhook and got results successfully !

PFB heroku Application logs (for shipping cost sample webhook):
2017-08-07T13:04:55.658984+00:00 heroku[web.1]: State changed from starting to crashed
2017-08-07T13:04:55.662081+00:00 heroku[web.1]: State changed from crashed to starting
2017-08-07T13:04:58.192041+00:00 heroku[web.1]: Starting process with command `python app.py`
2017-08-07T13:04:59.918427+00:00 app[web.1]:   File "app.py", line 56
2017-08-07T13:04:59.918441+00:00 app[web.1]:     print "Starting app on port %d" % port
2017-08-07T13:04:59.918446+00:00 app[web.1]:                                   ^
2017-08-07T13:04:59.918447+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-08-07T13:04:59.995529+00:00 heroku[web.1]: Process exited with status 1
2017-08-07T13:05:00.005672+00:00 heroku[web.1]: State changed from starting to crashed
2017-08-07T13:05:01.128651+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=shippingcostapi.herokuapp.com request_id=f17b5063-15b5-41e6-a675-49768fccd45e fwd="122.173.104.147" dyno= connect= service= status=503 bytes= protocol=https
2017-08-07T13:05:02.313134+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=shippingcostapi.herokuapp.com request_id=f9050b34-dc91-4017-bee3-a0dde69917e9 fwd="122.173.104.147" dyno= connect= service= status=503 bytes= protocol=https

**Comment by hemanth1991** [comment]
Yes even i faced the same issue.
If you closely look into the issue while deploying the app to Heroku -it happenns with Python 3.
In Python 3 Print is a function and should give with parenthesis.
i.e print("Starting app on port %d" % port).make the changes in app.py deploy to Heroku.
Test it --It works Fine :)
